scripts/cli/start_cli_testnet.sh

```diff
@@ -12,17 +12,28 @@ source "$HOME/.cargo/env"
 SCRIPT_PATH="$(dirname $0)"
 
 RUN_PARAMS="--host ac.testnet.libra.org --port 8000 -s $SCRIPT_PATH/consensus_peers.config.toml"
+RELEASE=""
 
-case $1 in
-    -h | --help)
-        print_help;exit 0;;
-    -r | --release)
-        echo "Building and running client in release mode."
-        cargo run -p client --release -- $RUN_PARAMS
-        ;;
-    '')
-        echo "Building and running client in debug mode."
-        cargo run -p client -- $RUN_PARAMS
-        ;;
-    *) echo "Invalid option"; print_help; exit 0;
-esac
+while true; do
```

Comment:
Does this  end up with an infinite loop? or do we need to do this while we still have params left to handle?

Reply:
It wouldn't loop infinitely, but it would wind up in the 'Invalid option' case erroneously when no args are passed.  The upcoming change addresses that.